site/collections/causes.php

```diff
@@ -8,6 +8,13 @@
 		'klimaliste-berlin',
 		'natucate',
 		'strassenfeger',
-		'teteenlair'
+		'teteenlair',
+		'klimadashboard',
+		'viseta',
+		'nowar',
+		'throughpositiveeyes',
+		'gobanyo',
+		'ppks',
+		'adunare'
```

Comment:
What do you think about using new `Cause: true` field instead static collection?

Reply:
I would still use `$kirby->collection('cases')` as entry but filtering that by a field, sure, makes sense to me.